playground/frontend/assets/thumb_down.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
One concern I have with these visual assets is how they were sourced and whether we have the rights to use them without attribution.  May we resolve this?

Reply:
All icons we creating from scratch, and these under apache licence.